packages/visual-editor/src/components/Dropzone/styles.module.css

```diff
@@ -7,7 +7,10 @@
   outline: 2px solid transparent;
   width: 100%;
   transition: outline 0.2s;
-  min-height: 120px;
+}
+
+.isExpanded {
+  min-height: 80px;
```

Comment:
Thoughts on adding something like `padding: 12px` to this drag state?

Reply:
I think it's a good suggestion and more padding could help with visualizing nested containers.

I tried some different `padding` and `margin` values here and unfortunately it throws off the drop zone positioning for some reason.  I went down the rabbit hole a little bit looking at `usePlaceholderStyle` but didn't see a way to fix the drop zone positions after `padding` or `margin` is added. 🤔 lets consider it for future updates

![Screenshot 2024-02-05 at 5 13 38 PM](https://github.com/contentful/experience-builder/assets/8539634/9f9e8592-494b-4668-91fa-fee0207248b7)